_posts/news/2025-01-07-announcing-scalajs-1.18.0.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: Announcing Scala.js 1.18.0
+category: news
+tags: [releases]
+permalink: /news/2025/01/07/announcing-scalajs-1.18.0/
+---
+
+
+We are pleased to announce the release of Scala.js 1.18.0!
+
+This release drops support for Scala 2.12.{2-5} and 2.13.{0-2}.
+Other than that, it is mostly a bugfix release.
+
+This release also contains a number of internal changes to the intermediate representation and linker.
+These are not externally visible, except for users who directly manipulate the IR and/or linker.
+They pave the way for upcoming changes that are still ongoing work.
+
+Read on for more details.
+
+<!--more-->
+
+## Getting started
+
+If you are new to Scala.js, head over to [the tutorial]({{ BASE_PATH }}/tutorial/).
+
+If you need help with anything related to Scala.js, you may find our community [in `#scala-js` on Discord](https://discord.com/invite/scala) and [on Stack Overflow](https://stackoverflow.com/questions/tagged/scala.js).
+
+Bug reports can be filed [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+## Release notes
+
+If upgrading from Scala.js 0.6.x, make sure to read [the release notes of Scala.js 1.0.0]({{ BASE_PATH }}/news/2020/02/25/announcing-scalajs-1.0.0/) first, as they contain a host of important information, including breaking changes.
+
+This is a **minor** release:
+
+* It is backward binary compatible with all earlier versions in the 1.x series: libraries compiled with 1.0.x through 1.17.x can be used with 1.18.0 without change.
+* It is *not* forward binary compatible with 1.17.x: libraries compiled with 1.18.0 cannot be used with 1.17.x or earlier.
+* It is *not* entirely backward source compatible: it is not guaranteed that a codebase will compile *as is* when upgrading from 1.17.x (in particular in the presence of `-Xfatal-warnings`).
+
+As a reminder, libraries compiled with 0.6.x cannot be used with Scala.js 1.x; they must be republished with 1.x first.
+
+## Changes with compatibility concerns
```

Comment:
I think we should mention that we run IR checking after the optimizer (maybe a more vulgarized form) and ask to report and explain how to turn IR checking off in case users get hit by this.